io/fv3atm_sfc_io.F90

```diff
@@ -27,27 +37,24 @@ module fv3atm_sfc_io
        Sfc_io_register_3d_fields, Sfc_io_copy_to_grid, Sfc_io_copy_from_grid, &
        Sfc_io_apply_safeguards, Sfc_io_transfer, Sfc_io_final
 
-  !> \defgroup fv3atm_sfc_io module
-  !> @{
-
-  !>@ Minimum temperature allowed for snow/ice
+  !> Minimum temperature allowed for snow/ice.
   real(kind=kind_phys), parameter :: timin = 173.0_kind_phys
 
   real(kind_phys), parameter:: min_lake_orog = 200.0_kind_phys
   real(kind_phys), parameter:: zero = 0, one = 1
 
   !> Internal data storage type for reading and writing surface restart files
   type Sfc_io_data_type
-    integer, public :: nvar2o = 0
-    integer, public :: nvar3 = 0
-    integer, public :: nvar2r = 0
-    integer, public :: nvar2mp = 0
-    integer, public :: nvar3mp = 0
-    integer, public :: nvar2l = 0
-    integer, public :: nvar2m = 0
-    integer, public :: nvar_before_lake = 0
-
-    ! The lsoil flag is only meaningful when reading:;
+    integer, public :: nvar2o = 0 !< ???
```

Comment:
@SamuelTrahanNOAA can you please tell us what the ??? in this documentation should be?

Reply:
Thanks @SamuelTrahanNOAA  for helping Larissa with this!